tailwind-annotator.config.mjs

```diff
@@ -0,0 +1,11 @@
+import tailwindConfig from './tailwind.config.mjs';
+
+export default {
+  presets: [tailwindConfig],
+  content: [
+    './src/annotator/components/**/*.js',
+    './node_modules/@hypothesis/frontend-shared/lib/**/*.js',
```

Comment:
The annotator CSS currently includes TW classes used by all frontend-shared components, though it doesn't use most of them. Do you have an idea as to how we could make sure that individual apps only ship the CSS for the components from the shared library they actually use? The same applies to a much lesser extent for the sidebar app.